dashboards/ram/ram_user_detail.sp

```diff
@@ -0,0 +1,334 @@
+        }
+      }
+
+    }
+
+  }
+
+  container {
+
+    title = "Alicloud RAM User Policy Analysis"
+
+    flow {
+      type  = "sankey"
+      title = "Attached Policies"
+      query = query.alicloud_ram_user_manage_policies_sankey
+      args = {
+        akas = self.input.user_aka.value
+      }
+
+      category "alicloud_ram_group" {
+        color = "ok"
+      }
+    }
+
+    table {
+      title = "Groups"
+      width = 6
+      query = query.alicloud_ram_groups_for_user
+      args = {
+        akas = self.input.user_aka.value
+      }
+
+      column "Name" {
+        // cyclic dependency prevents use of url_path, hardcode for now
+        href = "/alicloud_insights.dashboard.alicloud_ram_group_detail?input.group_title={{.'Name' | @uri}}"
+
+      }
+    }
+
+    table {
+      title = "Policies"
+      width = 6
+      query = query.alicloud_ram_all_policies_for_user
+      args = {
+        akas = self.input.user_aka.value
+      }
+    }
+
+  }
+}
+
+query "alicloud_ram_user_input" {
+  sql = <<-EOQ
+    select
+      title as label,
+      akas ->> 0 as value,
+      json_build_object(
+        'account_id', account_id
+      ) as tags
+    from
+      alicloud_ram_user
+    order by
+      title;
+  EOQ
+}
+
+query "alicloud_ram_user_mfa_for_user" {
+  sql = <<-EOQ
+    select
+      case when mfa_enabled then 'Enabled' else 'Disabled' end as value,
+      'MFA Status' as label,
+      case when mfa_enabled then 'ok' else 'alert' end as type
+    from
+      alicloud_ram_user
+    where
+      akas ->> 0 = $1
+  EOQ
+
+  param "akas" {}
+}
+
+query "alicloud_ram_user_direct_attached_policy_count_for_user" {
+  sql = <<-EOQ
+    select
+      coalesce(jsonb_array_length(attached_policy), 0) as value,
+      'Direct Attached Policies' as label,
+      case when coalesce(jsonb_array_length(attached_policy), 0) = 0 then 'ok' else 'alert' end as type
+    from
+      alicloud_ram_user
+    where
+     akas ->> 0 = $1
+  EOQ
+
+  param "akas" {}
+}
+
+query "alicloud_ram_user_overview" {
+  sql = <<-EOQ
+    select
+      name as "Name",
+      create_date as "Create Date",
+      user_id as "User ID",
+      akas ->> 0 as "ARN",
+      account_id as "Account ID"
+    from
+      alicloud_ram_user
+    where
+      akas ->> 0 = $1
+  EOQ
+
+  param "akas" {}
+}
+
+query "alicloud_ram_user_access_keys" {
+  sql = <<-EOQ
+    select
+      access_key_id  as "Access Key ID",
+      a.status as "Status",
+      a.create_date as "Create Date"
+    from
+      alicloud_ram_access_key as a left join alicloud_ram_user as u on u.name = a.user_name
+    where
+      u.akas ->> 0  = $1
+  EOQ
+
+  param "akas" {}
+}
+
+query "alicloud_ram_user_mfa_devices" {
+  sql = <<-EOQ
+    select
+      mfa ->> 'SerialNumber' as "Serial Number",
+      mfa ->> 'ActivateDate' as "Activate Date",
+      mfa -> 'User' ->> 'UserId' as "User Id"
+    from
+      alicloud_ram_user,
+      jsonb_array_elements(mfa_device_serial_number) as mfa
```

Comment:
@karanpopat mfa_device_serial_number is the text field; better we keep this in the overview table or try a single-column table